### ISSUE TITLE
Update peer dependency @phenomnomnominal/tsquery

### ISSUE THREAD

**Issue by jfai** [w11k/rx-ninja#18]
@phenomnomnominal/tsquery@4.1.0 allows TypeScript 4.

Without the update:

```
npm ERR! peer dep missing: typescript@^3, required by @phenomnomnominal/tsquery@4.0.0
```
```
$ npm list @phenomnomnominal/tsquery
`-- @w11k/rx-ninja@4.2.1
		`-- @phenomnomnominal/tsquery@4.0.0
```

Regards.

**Comment by romanroe** [comment]
We will take a look, thank you!

**Comment by romanroe** [comment]
@jfai Fixed in 4.2.2 🎉
